website/blog/2022-06-16-resources-migrating-your-react-native-library-to-the-new-architecture.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Resources - Migrating Your React Native Library to the New Architecture
+author: "cipolleschi"
```

Comment:
@cipolleschi You need to add yourself to `website/blog/authors.yml`

Reply:
thanks